sematic/db/models/factories.py

```diff
@@ -23,6 +23,7 @@
     value_from_json_encodable,
     value_to_json_encodable,
 )
+from sematic.types.types.union import _get_value_type, is_union
```

Comment:
Since `_get_value_type` is used outside of its own module now, we need to remove the leading `_`.

Reply:
Okay I will do that.